source/includes/_cluster.md

```diff
@@ -0,0 +1,419 @@
+`consul` | dict | Detailed state of the consul cluster being part of the manager infrastructure.
+`error` | string | Description of a fatal error that occured during cluster configuration or operation, if any.
+`logs` | list | Logs of the cluster operations on the current node.
+
+## Get Cluster State
+
+> Request Example
+
+```shell
+$ curl -u user:password "http://<manager-ip>/api/v3/cluster"
+```
+
+```python
+client.cluster.status()
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/",
+  "method": "GET",
+  "headers": headers
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```html
+obsolete
+```
+
+> Response Example (cluster not initialized)
+
+```json
+{
+    "initialized": false
+}
+```
+
+> Response Example
+
+```json
+{
+    "initialized": true,
+    "consul": {
+        "leader": "172.20.0.3:8300"
+    },
+    "encryption_key": "<REDACTED>",
+    "error": null,
+    "logs": [
+        {
+            "message": "HA Cluster configuration complete",
+            "timestamp": 1485778546965628,
+            "cursor": "opaque cursor value"
+        }
+    ]
+}
+```
+
+`GET "{manager-ip}/api/v3/cluster"`
+
+Gets the current cluster state. The `logs` and `error` fields are hidden by
+default, but can be added to the response if specified using the `_include`
+query parameter.
+
+### Response
+A `ClusterState` resource.
+
+### URI Parameters
+* `since`: When including logs, fetch only logs after this cursor value.
+
+
+## Put Cluster State
+
+> Request Example
+
+```python
+client.cluster.start(
+    host_ip='172.20.0.2',
+    node_name='manager',
+    encryption_key='<REDACTED>'
+)
+```
+
+```javascript
+var headers = {
+   'content-type': 'application/json',
+   'authorization': 'Basic ' + new Buffer(username + ':' + password).toString('base64')
+}
+
+var settings = {
+  "url": "http://<manager-ip>/api/v3/cluster/",
+  "method": "PUT",
+  "headers": headers,
+  "contentType": "application/json"
+  "data": JSON.stringify({
+      "host_ip": "172.20.0.2",
+      "node_name": "manager",
+      "encryption_key": "<REDACTED>"
+  })
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+```
+
+```shell
+$ curl -X PUT -u user:password -H "Content-Type: application/json" -d '{"host_ip": "172.20.0.2", "node_name": "manager", "encryption_key": "<REDACTED>"}' http://<manager-ip>/api/v3/cluster
+```
+
+
+```html
+obsolete
+```
+> Response Example
+
+```json
+{
+    "initialized": false
+}
+```
+
+`PUT "{manager-ip}/api/v3/cluster"`
+
+Starts the cluster mechanisms on the current manager: either joins an existing
+cluster, if the `join_addrs` parameter is provided, otherwise bootstraps a new
+cluster.
```

Comment:
Starts the cluster mechanisms on the current ~manager~ __Cloudify Manager.__ __If the `join_addrs` parameter is provided, joins an existing cluster,__ otherwise bootstraps a new cluster.